src/app/children/child-details/child-details.component.ts

```diff
@@ -80,7 +83,7 @@ export class ChildDetailsComponent implements OnInit {
       motherTongue:   [{value: this.child.motherTongue,   disabled: !this.editing}],
       religion:       [{value: this.child.religion,       disabled: !this.editing}],
 
-      center:         [{value: this.child.center,         disabled: !this.editing}],
+      center:         [{value: this.child.center,         disabled: !this.editing}, Validators.required],
```

Comment:
why exaclty is the center required?

Reply:
You've got to ask @sleidig on this one...

Reply:
as we use the `center` for basic filters and grouping it somewhat makes sense to require it. we could also keep it optional as well, however.